conf/messages

```diff
@@ -102,10 +105,10 @@ contact.form.specific.query=If you have a specific tax query {0}.
 contact.form.please.fill=If you need help using your tax account please fill in this form.
 contact.form.how.help=How can we help you?
 form.send=Send
-contact.name.label=Your name
+contact.name.label=Full name
 contact.name.error.required=Enter your name
```

Comment:
I think these error messages will need updating to be consistent i.e. 'Enter your full name'

Reply:
@matthewmascord Good suggestion, error messages updated.